### PR TITLE
GRPC V2 : Get Election Info

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for grpc v2 `GetElectionInfo`

## Changes
Added support for grpc v2 `GetElectionInfo`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetElectionInfo.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetElectionInfo", mixinStandardHelpOptions = true)
+public class GetElectionInfo implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20002")
```
Comment (abizjak) [inline concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetElectionInfo.java:25]: I would recommend using 20000 as the port. 20002 is not a default anywhere, so setting it to that is not really that useful.

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/election/ElectionInfoBaker.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.transactions.AccountAddress;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Builder
+@ToString
+@EqualsAndHashCode
+public class ElectionInfoBaker {
```
Comment (abizjak) [inline concordium-sdk/src/main/java/com/concordium/sdk/responses/election/ElectionInfoBaker.java:12]: Some documentation please.

**Review by abizjak — COMMENTED** [review]
Add documentation. Otherwise looks good.

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetElectionInfo.java (hunk 1 above)
Comment (parv0888) [reply concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetElectionInfo.java:25]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/election/ElectionInfoBaker.java (hunk 2 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/responses/election/ElectionInfoBaker.java:12]: done

**Comment by EmilLa1** [comment]
GJ @parv0888. Please merge and it's good to go.

**Review by EmilLa1 — APPROVED** [review]

**Pull request merged**